### PR TITLE
Update README.md adding instructions to help the user to connect on bastion host

### PR DESCRIPTION
**Pull request by walves-cit**
Updates in the README file:

- Adding instructions to help the user to connect on bastion host. How to be allowlisted.

### CODE REVIEW

**Comment by walves-cit** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
>  **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fterraform-example-foundation-app%2Fpull%2F51) for more info**.

@googlebot I signed it!

**Comment by walves-cit** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
>  **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fterraform-example-foundation-app%2Fpull%2F51) for more info**.

@googlebot I signed it!

**Comment by walves-cit** [comment]
@googlebot I signed it!

**Comment by walves-cit** [comment]
@googlebot I signed it!

File: 6-anthos-install/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Deploying Bank of Anthos
-These instructions need to be run on the bastion host. To access the bastion host, open the Google Cloud console, and navigate to Compute Engine in the GKE project. In my instance, `prj-bu1-d-boa-gke` Then, select the `gce-bastion-us-west1-b-01` and click on `ssh`. You need to be a whitelisted member to access this node.
+Before you begin, you need to be able to access the bastion host using IAP TCP forwarding. Follow these [instructions](https://cloud.google.com/iap/docs/using-tcp-forwarding#grant-permission) to grant this access to your user.
```
Comment (bharathkkb) [inline 6-anthos-install/README.md:2]: We should expose `bastion_members` from here https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/ae238f3cce0f9541e683ec14bb2f23d307e51354/5-infrastructure/modules/base-env/variables.tf#L73 in each env.

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @walves-cit

File: 6-anthos-install/README.md (hunk 1 above)
Comment (rutalreja-deloitte) [reply 6-anthos-install/README.md:2]: @bharathkkb addressed in #50 this PR can be closed

**Pull request closed without merging**